src/stores/user.ts

```
@@ -0,0 +1,44 @@
+    async fetchLoggedInUser() {
+      this.error = null;
+      try {
+        const authStore = useAuthStore();
+        
+        if (!authStore.accessToken) {
+          throw new Error('Access token is missing');
+        }
+        const user = await UserService.getLoggedInUser(authStore.accessToken);
+        this.currentUser = user;
+        return user;
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)